projects/ngrx.io/content/guide/store/recipes/injecting.md

```diff
@@ -56,12 +56,12 @@ export class FeatureModule {}
 
 ## Injecting Meta-Reducers
 
-To inject 'middleware' meta reducers, use the `META_REDUCERS` injection token exported in
+To inject 'middleware' meta reducers, use the `USER_PROVIDED_META_REDUCERS` injection token exported in

```

Comment:
We want to keep this to `META_REDUCERS`.

The `META_REDUCERS` token provides a way for library owners to add meta-reducers.

Reply:
@timdeschryver Oh cool, understood, I'll resolve this!

Reply:
@timdeschryver I have rolled back my change per what you've said.